k8s/perftest/common/kube-system/disable-ipv6.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    component: ds-disable-ipv6
+  name: ds-disable-ipv6
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      component: ds-disable-ipv6
+      tier: node
+  template:
+    metadata:
+      labels:
+        component: ds-disable-ipv6
+        tier: node
+    spec:
+      containers:
+        - command:
+            - nsenter
+            - --target
+            - "1"
+            - --mount
+            - --uts
+            - --ipc
+            - --net
+            - --pid
+            - --
+            - sh
+            - -c
+            - |
+              if ip a | grep "inet6"; then
+                echo ipv6 is not disabled
+                if cat /etc/default/grub | grep "GRUB_CMDLINE_LINUX_DEFAULT=" | grep "ipv6.disable=1"; then
+                  echo GRUB_CMDLINE_LINUX_DEFAULT is already disabled
+                else
+                  echo disabling GRUB_CMDLINE_LINUX_DEFAULT
+                  sed -i 's/\bGRUB_CMDLINE_LINUX_DEFAULT="/GRUB_CMDLINE_LINUX_DEFAULT="ipv6.disable=1 /g' /etc/default/grub
+                fi
+                if cat /etc/default/grub | grep "GRUB_CMDLINE_LINUX=" | grep "ipv6.disable=1"; then
+                  echo GRUB_CMDLINE_LINUX is already disable
+                else
+                  echo disabling GRUB_CMDLINE_LINUX
+                  sed -i 's/\bGRUB_CMDLINE_LINUX="/GRUB_CMDLINE_LINUX="ipv6.disable=1 /g' /etc/default/grub
+                fi
+                update-grub
+                # reboot
```

Comment:
Shouldn’t this be enabled?

Reply:
No, reboot after draining now. Add to knode later. That's what we agreed.